lib/ingestors/ingestor_event_rest.rb

```diff
@@ -1014,8 +1014,9 @@ def process_maastricht(url)
           event.description = convert_description ical_event.description
           event.url = ical_event.url
           # TeSS timezone handling is a bit special.
-          event.start = ical_event.dtstart
-          event.end = ical_event.dtend
+          # remove the timezone shift since else TeSS will shift it too much
+          event.start = ical_event.dtstart.to_s.split[0,2].join(' ').to_s_to_time
```

Comment:
I'd never heard of the `to_s_to_time` method, fancy. Can you make a utility method to strip off the timezone from a datetime instead of doing this inline?

Reply:
and test it.